hash/ekara_japan_d.xml

```diff
@@ -92,4 +92,32 @@ license:CC0-1.0
 		</part>
 	</software>
 
+	<software name="dc0006">
+		<description>Dream Cartridge: Best Hit Vol.10 (Japan) (DC0006-BHT)</description>
+		<year>2004</year>
+		<publisher>Takara</publisher>
+		<info name="alt_title" value="ドリームカートリッジ ベストヒット Vol.10"/> <!-- from cover art -->
+		<info name="alt_title" value="ベストヒットvol.10"/> <!-- shown on-screen when run on Popira -->
+		<part name="cart" interface="ekara_cart">
+			<feature name="slot" value="rom_24lc02"/>
+			<dataarea name="rom" size="0x100000">
+				<rom name="dc0006-bht.u1" size="0x100000" crc="62368c46" sha1="fdedb4b81c3b1642eb3a8993cdcf58dbc29f9a03"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="dc0008">
+		<description>Dream Cartridge: Best Artist Vol.7 (Japan) (DC0008-BAT)</description>
+		<year>2004</year>
+		<publisher>Takara</publisher>
+		<info name="alt_title" value="ドリームカートリッジ ベストアーティスト Vol.7"/> <!-- from cover art -->
+		<info name="alt_title" value="ドリームカートリッジvol.7"/> <!-- shown on-screen when run on Popira -->
+		<part name="cart" interface="ekara_cart">
+			<feature name="slot" value="rom_24lc02"/>
+			<dataarea name="rom" size="0x100000">
+				<rom name="dc0008-bat.u1" size="0x100000" crc="b97472a5" sha1="681990c795094cf83b200c947ed34624d1fc90b3"/>
+			</dataarea>
+		</part>
+	</software>
```

Comment:
Have you confirmed the titles displayed on Popira for these two (not just copy/pasted comments from other items in this software list)?  I ask because I recall in one of the e-kara lists that some cartridges didn’t use the same title format as the others in their series.

Reply:
Can you post screenshots or ask someone?  It’s far more of a pain to track down and fix bad info in software lists than it is to sort it out *before* it goes in.